### PR TITLE
[ID] Update GMT, NAT, and BanchoBot

### PR DESCRIPTION
**Pull request by hikikuman**
Based on references from: https://github.com/ppy/osu-wiki/pull/4828

Except Reporting Bad Behaviour articles, which can be found in this PR: https://github.com/ppy/osu-wiki/pull/4877

### CODE REVIEW

File: wiki/People/The_Team/Account_support_team/id.md [hunk 1]
```diff
@@ -25,17 +20,24 @@ Kamu harus menghubungi tim ini mengenai topik yang berhubungan dengan akun yang
...
 - Perilaku buruk yang [ingin kamu akui](/wiki/Reporting_Bad_Behaviour/Handling_Foul_Play#what-can-i-do-if-i've-broken-the-rules?).
 - [Kehilangan akses ke email](/wiki/Help_Centre#sign-in) yang terhubung dengan akun osu! kamu, atau saat akun kamu dicuri.
 - [Mengembalikan nama pengguna menjadi yang sebelumnya dan sedikit koreksi ejaan pada nama pengguna](/wiki/Help_Centre#name-changes).
 - [Pendaftaran akun bot](/wiki/Bot_Account).
 
 ### [support@ppy.sh](mailto:support@ppy.sh)
 
+*Catatan: Untuk melaporkan sebuah kasus seperti perilaku buruk dan pelecehan seksual yang ekstrim dari pengguna mana pun, termasuk dari seorang moderator, silakan kunjungi artikel [Melaporkan Penyalahgunaan](/wiki/Reporting_Bad_Behaviour/Abuse).*
+
+- Kekhawatiran terhadap [perilaku dari seorang moderator](/wiki/People/The_Team/Global_Moderation_Team#staff-complaints).
```
Comment (AncuL001) [inline wiki/People/The_Team/Account_support_team/id.md:37]:
```suggestion
- Kekhawatiran terhadap perilaku buruk dari seorang moderator.
```
seems like the en translation doesn't have the link thing

Comment (AncuL001) [inline wiki/People/The_Team/Account_support_team/id.md:37]:
```suggestion
- Kekhawatiran terhadap perilaku buruk dari seorang moderator.
```
the english ver doesn't have the hyperlink thing

**Review by AncuL001 — CHANGES_REQUESTED** [review]
this is the only one i could find, the rest is clean

though the GMT page is gonna be outdated soon with the addition of Niva

File: wiki/People/The_Team/Account_support_team/id.md (hunk 1 above)
Comment (hikikuman) [reply wiki/People/The_Team/Account_support_team/id.md:37]: eh, didn't notice that one. thanks!

**Comment by hikikuman** [comment]
@AncuL001 yea, also osu! Alumni since Niva was part of alumni before he became a GMT. Anyway, can you recheck these articles again? Hope that nothing is missed. Thank you!

**Review by AncuL001 — DISMISSED** [review]

File: wiki/People/The_Team/Account_support_team/id.md [hunk 2]
```diff
@@ -25,17 +20,24 @@ Kamu harus menghubungi tim ini mengenai topik yang berhubungan dengan akun yang
 
 ### [accounts@ppy.sh](mailto:accounts@ppy.sh)
 
-- [Banding atas pembatasan yang diberlakukan pada akun kamu](/wiki/Help_Centre/Account_Restrictions), termasuk silence dan penghapusan konten profil.
+- [Banding atas pembatasan yang diberlakukan pada akun kamu](/wiki/Help_Centre/Account_Restrictions), termasuk:
```
Comment (Nivalyx) [inline wiki/People/The_Team/Account_support_team/id.md:23]: iirc all instances of `pembatasan` in the osu! website has been reverted back to `restriction` for convenience-related purposes tho. 

if you want to keep the `pembatasan` here you might want to go for sth like `Banding atas pembatasan (*restriction*) yang diberlakukan`

image

File: wiki/People/The_Team/Account_support_team/id.md [hunk 3]
```diff
@@ -25,17 +20,24 @@ Kamu harus menghubungi tim ini mengenai topik yang berhubungan dengan akun yang
...
+  - penghapusan beatmap;
+  - penghapusan postingan dan komentar di forum.
 - Perilaku buruk yang [ingin kamu akui](/wiki/Reporting_Bad_Behaviour/Handling_Foul_Play#what-can-i-do-if-i've-broken-the-rules?).
 - [Kehilangan akses ke email](/wiki/Help_Centre#sign-in) yang terhubung dengan akun osu! kamu, atau saat akun kamu dicuri.
 - [Mengembalikan nama pengguna menjadi yang sebelumnya dan sedikit koreksi ejaan pada nama pengguna](/wiki/Help_Centre#name-changes).
 - [Pendaftaran akun bot](/wiki/Bot_Account).
 
 ### [support@ppy.sh](mailto:support@ppy.sh)
 
+*Catatan: Untuk melaporkan sebuah kasus seperti perilaku buruk dan pelecehan seksual yang ekstrim dari pengguna mana pun, termasuk dari seorang moderator, silakan kunjungi artikel [Melaporkan Penyalahgunaan](/wiki/Reporting_Bad_Behaviour/Abuse).*
```
Comment (Nivalyx) [inline wiki/People/The_Team/Account_support_team/id.md:35]: `sebuah kasus` --> `kasus-kasus`

(itu contohnya ada lebih dari 1 soalnya)

File: wiki/People/The_Team/Global_Moderation_Team/id.md [hunk 4]
```diff
@@ -3,38 +3,43 @@ tags:
...
 3. Memiliki pemahaman yang cukup tentang [peraturan](/wiki/Rules) serta penerapannya. Penting untuk mengetahui apa saja yang dapat diterima dan tidak diterima di osu!
 4. Bersabarlah. GMT tidak menambahkan secara spontan anggota yang telah membantu hanya beberapa hari. Butuh waktu bagi GMT untuk mengevaluasi seseorang⁠ — kepribadian mereka, apa yang dapat mereka bantu, aktivitas mereka, dll.
 
-Umumnya, tim akan mengamati orang selama beberapa minggu untuk memastikan bahwa mereka layak ditambahkan ke tim. Harap bersabar dan jangan berlebihan. Jangan ragu untuk bertanya kepada siapa pun dari GMT tentang pertanyaan apa pun terkait peraturan, melaporkan pengguna, dan masalah lainnya.
+Umumnya, tim akan mengamati orang selama beberapa minggu atau bulan untuk memastikan bahwa mereka layak ditambahkan ke tim. Harap bersabar dan jangan berlebihan. Jangan ragu untuk bertanya kepada siapa pun dari GMT tentang pertanyaan apa pun terkait peraturan, melaporkan pengguna, dan masalah lainnya.
```
Comment (Nivalyx) [inline wiki/People/The_Team/Global_Moderation_Team/id.md:42]: `orang` --> `kandidat-kandidat yang dainggap layak` maybe? 

`orang` feels... out of touch here :(

File: wiki/People/The_Team/Account_support_team/id.md (hunk 3 above)
Comment (hikikuman) [reply wiki/People/The_Team/Account_support_team/id.md]: oke, diubah. kalo yang lainnya itu kaya yang di line 9 kah? `anggotanya` jadi `anggota-anggotanya` atau `masing-masing anggota`? udah cek di semua artikel, nyoba2 ctrl+F `sebuah` atau `suatu` atau `satu`, tapi nggak nemu yang lainnya lagi, dan udah cek ulang juga di masing2 artikel. takutnya ada yang miss D:

File: wiki/People/The_Team/Account_support_team/id.md (hunk 2 above)
Comment (hikikuman) [reply wiki/People/The_Team/Account_support_team/id.md]: Changed `pembatasan` to `restriction` on line 5 and 17. I'll take a look for this article https://osu.ppy.sh/wiki/id/Help_Centre/Account_Restrictions later, as well as in other articles that contain restrictions.

File: wiki/People/The_Team/Account_support_team/id.md (hunk 3 above)
Comment (Nivalyx) [reply wiki/People/The_Team/Account_support_team/id.md]: imo semuanya applicable kok, tergantung konteks aja hehe

kalau yang dicontohkan itu cuma satu pakainya `sebuah (x)` / `suatu (x)` / `satu (x)`
kalau yang dicontohkan ada lebih dari satu pakainya `para anggotanya` / `seluruh anggotanya` / `anggota-anggotanya` / dsb

untuk line 9 itu sendiri coba pakai kalimat begini deh :

``Tim ini **hanya** dapat dihubungi melalui email pada alamat [support@ppy.sh](mailto:support@ppy.sh), [accounts@ppy.sh](mailto:accounts@ppy.sh), atau [privacy@ppy.sh](mailto:privacy@ppy.sh). Identitas para anggotanya sengaja dirahasiakan untuk melindungi mereka semua dari hal-hal yang tidak diinginkan.``

Comment (Nivalyx) [reply wiki/People/The_Team/Account_support_team/id.md]: also uhh `Point of Contact` nggak literally berarti `Titik Kontak` juga orz :(

mending dijadiin `Kontak` aja itu imo...

Comment (hikikuman) [reply wiki/People/The_Team/Account_support_team/id.md]: hokkeee, all applied. udah recheck dan ada sedikit perubahan di paragraf awal2. thank you buat saran dan masukannya niva!

File: wiki/People/The_Team/Global_Moderation_Team/id.md [hunk 5]
```diff
@@ -3,38 +3,43 @@ tags:
...
 
 **Semua Global Moderator bersumpah untuk menjunjung tinggi [Kode Etik Kontributor](/wiki/Contributor_Code_of_Conduct) sebagai tambahan dari [Peraturan Komunitas](/wiki/Rules).**
 
-Jika kamu menemukan anggota Global Moderation Team yang bertindak tidak senonoh, harap segera hubungi [abuse@ppy.sh](mailto:abuse@ppy.sh) dengan disertai informasi yang selengkap-lengkapnya. Untuk informasi lebih lanjut, harap lihat [Melaporkan Penyalahgunaan](/wiki/Reporting_Bad_Behaviour/Abuse).
-
 ## Peran dan Tanggung Jawab
 
-Global Moderation Team bertanggung jawab atas kesejahteraan obrolan dan forum, dan mengurus interaksi di dalam komunitas. Mereka harus didekati jika Anda memiliki pertanyaan yang berkaitan dengan aspek moderasi permainan, yang akan mencakup (namun tidak terbatas pada) masalah dengan obrolan (silences, pelecehan) and di forum (spam).
+***Catatan: Untuk melaporkan seorang moderator yang berperilaku buruk atau melanggar Kode Etik Kontributor, silakan hubungi [account support team](/wiki/People/The_Team/Account_support_team#support@ppy.sh).***\
+*Lihat juga: [Melaporkan Penyalahgunaan](/wiki/Reporting_Bad_Behaviour/Abuse).*
```
Comment (TPGPL) [inline wiki/People/The_Team/Global_Moderation_Team/id.md:20]:
```suggestion
*Lihat juga: [Melaporkan Penyalahgunaan](/wiki/Reporting_Bad_Behaviour).*
```

**Review by TPGPL — DISMISSED** [review]

File: wiki/BanchoBot/id.md [hunk 6]
```diff
@@ -6,8 +6,6 @@ tags:
   - announcements
   - perintah
   - pengumuman
-outdated: true
-outdated_since: 83725ce048670f4ac909cf76e228a9a5b792df4a
 ---
 
 # BanchoBot
```
[the remark this answers is not in the document]
Comment (hikikuman) [reply wiki/BanchoBot/id.md:11]: Sorry, my bad. Added missing translation in `Where` section.

**Review by Nivalyx — APPROVED** [review]
translation-wise everything looks okay now

**Review by TicClick — APPROVED** [review]

**Pull request merged**